xunit-autolabeler-v2/ast_parser/lib/constants.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2020 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import re
+import os
+
+
+IGNORED_METHOD_NAMES = (
+    'run_command',
+    'parse_command_line_args',
+    'main'
+)
+
+
+__APPENGINE_LIB_REGEX = re.compile('appengine/(.+/)*lib')
+
+
+START_VERB_REGEX = re.compile('\\[START\\s(\\w+)\\]')
+
+
+def REGION_TAG_PREDICATE(path):
```

Comment:
Done - though I was treating this as a "constant" previously, since I didn't want it to be buried too deep in the codebase (hence `CONSTANT_CASE`).

...but if we're making this a function though - would it make more sense to relocate it closer to its use in `get_region_tags`? (even if that makes it harder to find?)